Library/Homebrew/unpack_strategy/tar.rb

```diff
@@ -38,9 +38,10 @@ def extract_to_dir(unpack_dir, basename:, verbose:)
         end
 
         system_command! "tar",
-                        args:    ["xf", tar_path, "-C", unpack_dir],
+                        args:    ["xf", tar_path, "-C", unpack_dir, *extra_args],
                         verbose: verbose
       end
     end
   end
 end
+require "extend/os/linux/unpack_strategy/tar" if OS.linux?
```

Comment:
Please read my messages so I don't have to keep restating them:

1. You need to add an empty newline before the `require`
2. You need to `require "extend/unpack_strategy/tar"`. Do not do `if OS.linux?` in this file.
3. You need to create a new file `extend/unpack_strategy/tar` which has `require "extend/os/linux/unpack_strategy/tar" if OS.linux?` in it.

Do not mark this as resolved until I do so myself, please.

Reply:
Ok, fine.